sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -585,12 +585,13 @@ INSERT INTO `ccda_sections` (`ccda_sections_id`, `ccda_components_id`, `ccda_sec
 #EndIf
 
 #IfNotRow2D list_options list_id lists option_id Care_Team_Status
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('lists', 'Care_Team_Status', 'Care Team Status', '1');
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','active','Active',10,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','inactive','Inactive',20,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','suspended','Suspended',30,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','proposed','Proposed',40,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','entered-in-error','Entered In Error',50,0,0);
+INSERT INTO list_options (list_id,option_id,title, seq, is_default, option_value, notes) VALUES ('lists','Care_Team_Status','Care Team Status',0, 1, 0, 'This list originally comes from http://hl7.org/fhir/R4/valueset-care-team-status.html');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','proposed','Proposed',10,0,1, 'The care team has been drafted and proposed, but not yet participating in the coordination and delivery of patient care.');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','active','Active',20,1,1, 'The care team is currently participating in the coordination and delivery of care.');
```

Comment:
@adunsulag ,
Are you meaning to have active selected by default. This means whenever make a new patient this will be set to Active even if no care team provider(s) or facility(s are selected)?

Reply:
i ended up removing this default setting for now in the followup PR since it was really odd to have in testing (ie. whenever make a new patient or edit a patient, it will default to active even though most users will never see or edit this when editing the demographics).

Reply:
I've been defaulting list options so the user doesn't have to think about what to select and we can provide a value via FHIR.  It makes sense to remove the default here as if there is no care team we don't want a status.  I guess if we wanted to we could add javascript (or on the backend on save) to require the status to be selected if care team is provided.

If there is a CareTeam record we are supposed to have a value for the status.  If there is no value we report it as an unknown code.  According to US Core FHIR spec the unknown code and Data Absent Reason (DAR) is only supposed to be used for legacy data and when importing data from other systems where the value is not known.

Reply:
actually an active status for no careteam would be correct imo.

Reply:
if a user assigns a provider or facility in care team and doesn't set the status (ie. keeps is unassigned), can't we just assume it is 'Active' when used downstream in fhir etc.?

Reply:
placed a PR that reverts the active to be the default setting (since guessing fhir needs to both support active and unassigned(unknown code)), which also will place active for all patients during the ugprade (one time thing):
https://github.com/openemr/openemr/pull/4565
(just let me know if want to bring this in and then I'll bring it in)